topdown/object_get.go

```diff
@@ -0,0 +1,23 @@
+package topdown
+
+import (
+	"github.com/open-policy-agent/opa/ast"
+	"github.com/open-policy-agent/opa/topdown/builtins"
+)
+
+func builtinObjectGet(a, b, c ast.Value) (ast.Value, error) {
+	object, err := builtins.ObjectOperand(a, 1)
+	if err != nil {
+		return nil, err
+	}
+
+	if ret := object.Get(&ast.Term{Value: b}); ret != nil {
+		return ret.Value, nil
+	}
+
+	return c, nil
+}
+
+func init() {
+	RegisterFunctionalBuiltin3(ast.ObjectGet.Name, builtinObjectGet)
```

Comment:
[isn't this deprecated?](https://github.com/open-policy-agent/opa/blob/089a19120810b504b2f419908c54be3d24b4ae14/topdown/builtins.go#L64) I haven't dug in just now, but if so, this should better use the other function, I suppose :)

Reply:
OK, cool, I'll change it accordingly.

Reply:
Done.